int32 idx_a
int32 idx_b
---
